src/main/resources/templates/HumanReadable.liquid

```diff
@@ -2,337 +2,420 @@
 <html>
```

Comment:
I think we have templates in FHIR service and packaging utility. is this template still needed?

Reply:
Removed.